### DISCUSSION TITLE
Option to disable mod score multiplier in multiplayer

### DISCUSSION THREAD

**Discussion by timiimit** [ppy/osu#23452] [Ideas]
Multi is often just played for fun. Some people pick certain freemods just because it is their preferred way of playing, not specifically to gain an advantage (or disadvantage) over others. This would allow people with varying skill to compete on equal terms. In the background scoring would still be handled normally, score multiplier would only be disabled visually.

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/20531#discussioncomment-3786492

**Comment by timiimit** [comment]
Oh, i missed that comment. Would it be okay if i work on implementing this?

**Reply by bdach** [reply]
I wouldn't recommend doing that as a first contribution. It's going to be an involved task spanning multiple systems.
